include/cpu_features_macros.h

```diff
@@ -359,6 +363,9 @@
 #endif
 #endif  //  defined(CPU_FEATURES_ARCH_RISCV)
 
+#if defined(CPU_FEATURES_ARCH_LOONGARCH)
```

Comment:
This is unused, let's remove it for now.

Reply:
OK, fixed.